projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1058,6 +1066,36 @@ public static function notice( $message, $status = 'info', $classes = '' ) {
 		);
 	}
 
+	/**
+	 * Retrieve site-specific features for Simple sites.
+	 *
+	 * We're caching the data for the lifetime of the request, because it can be slow to calculate,
+	 * and it can be called multiple times per single request.
+	 *
+	 * We intentionally don't use object caching or any other type of persistent caching,
+	 * in order to avoid complex cache invalidation on subscription addition or removal.
+	 *
+	 * @since 10.7
+	 *
+	 * @return array
+	 */
+	public static function get_site_specific_features() {
```

Comment:
Does this need to be public?
```suggestion
	private static function get_site_specific_features() {
```